Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/web/WebComposeFeedFragment.kt

```diff
@@ -20,9 +26,32 @@ class WebComposeFeedFragment : Fragment() {
     ): View = ComposeView(requireContext()).apply {
         setContent {
             // Add Taboola web feed to your current WebView
-            CustomerWebView(PlacementInfo.webFeedProperties())
+            CustomerWebView(PlacementInfo.webFeedProperties(), taboolaWebWrapperViewModel)
         }
     }
 }
 
+/**
+ * This method adds WebView using Compose and integrate with Taboola SDK (Web Integration)
+ */
+@Composable
+fun CustomerWebView(
+    placementInfo: PlacementInfo.WebFeedProperties,
+    taboolaWebWrapperViewModel: TaboolaWebWrapperViewModel
+) {
+    //This is the publisher code which displays a WebView on the screen
+    AndroidView(factory = { context ->
+        WebView(context).apply {
+            layoutParams = ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.MATCH_PARENT,
+                ViewGroup.LayoutParams.MATCH_PARENT
+            )
+
+
+            taboolaWebWrapperViewModel.setupWebViewForTaboola(this)
```

Comment:
@voltfmanSlava  - Why is the setup for the viewModel happening inside the WebView composable?

Reply:
@tomer-br-taboola because i need webview to be passed to taboolaWebWrapperViewModel.